vertical-pod-autoscaler/pkg/admission-controller/certs.go

```diff
@@ -99,7 +98,7 @@ func NewKeypairReloader(config certsConfig) (*KeypairReloader, error) {
 
 				// watch for errors
 			case err := <-watcher.Errors:
-				klog.Infof("error", err)
+				klog.Infof("error: %v", err)
```

Comment:
We need more descriptive log here. Probably also warning or error level. 
Is there any action to be taken on observing errors from the watcher?